### ISSUE TITLE
Does this work with SecureCRT v9 now?

### ISSUE THREAD

**Issue by GD-Dal** [PresidioCode/SecureCRT#57]
Thought I read some place in the docs that this isn't working with SecureCRT v9.
Is this still true?

Thanks

**Comment by jamiecaesar** [comment]
Yes, that is still true.  Unfortunately my current projects don't have me using SecureCRT and so I haven't had enough free time to get the new version and figure out how much things have changed to get the scripts working in v9.  Sorry!

**Comment by MrMoosieMan** [comment]
How about now?  :)
